corehq/apps/domain/views.py

```diff
@@ -3183,6 +3183,58 @@ def page_context(self):
         }
 
 
+class Dhis2LogListView(BaseAdminProjectSettingsView, ListView):
```

Comment:
Any reason you chose to use this and not `GenericTabularReport` e.g. https://github.com/dimagi/commcare-hq/blob/898e161ef3ef375d573fac58612027a53db1fb74/corehq/apps/domain/views.py#L2268-L2268